addons/sale/static/src/js/sale_portal.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module */
+
+import publicWidget from 'web.public.widget';
+
+publicWidget.registry.PortalHomeCounters.include({
+    /**
+     * @override
+     */
+    getCountersToKeep() {
+        let countersToKeep = this._super(...arguments);
+        return countersToKeep.concat(['quotation_count', 'order_count']);
```

Comment:
Sugg:
`return this._super(...arguments).concat(['quotation_count', 'order_count']);`

(Same for account override).